packages/@lightningjs/ui-components-test-utils/src/lightning-test-renderer.js

```diff
@@ -29,8 +29,11 @@ context.setTheme({
 const stage = {
   w: 1920,
   h: 1080,
-  clearColor: 0x00000000,
-  canvas2d: false,
+  precision: 2 / 3,
```

Comment:
I'm a little confused by the default here. I thought if the precision is 2/3, then the w/h should be 1280x720 (like our Storybook setup). Otherwise, if you actually try testing this setup in Storybook, you'll see that the stage size is full size, but the content on it is shrunk down.

Reply:
That makes sense for both to match. Updated the TestRenderer stage to match what is used in the storybook config here: 8786de0ed99bfe5c58b843dd84a57b1e3ab2deea